### PR TITLE
Add support for Debian 9

### PR DESCRIPTION
**Pull request by hdep**
Hello,

here is a PR for adding support for Debian 9.
Please let me know if it is ok. Currently working great on my system.

Hugo

### CODE REVIEW

**Comment by hdep** [comment]
Hello, I don't understand why the build failed, some are OK other are not, any help appreciated @ghoneycutt

File: manifests/init.pp [hunk 1]
```diff
@@ -916,42 +916,68 @@
...
           case $::lsbmajdistrelease {
             /(7|8)/: {
-
               if $ensure_vas == 'present' {
                 fail("Pam: vas is not supported on ${::osfamily} ${::lsbmajdistrelease}")
               }
               $default_pam_d_login_template = "pam/login.debian${::lsbmajdistrelease}.erb"
               $default_pam_d_sshd_template  = "pam/sshd.debian${::lsbmajdistrelease}.erb"
               $default_package_name         = 'libpam0g'
-
```
Comment (ghoneycutt) [inline manifests/init.pp]: These lines should stay for readability

File: templates/login.debian9.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+auth       optional   pam_faildelay.so  delay=3000000
```
Comment (ghoneycutt) [inline templates/login.debian9.erb]: is this data for a default, vanilla install of debian9?

Comment (hdep) [reply templates/login.debian9.erb]: I just checked on another VM, yes it does.

File: manifests/init.pp (hunk 1 above)
Comment (hdep) [reply manifests/init.pp]: Fixed in last commit

**Comment by hdep** [comment]
Hello @ghoneycutt can I do something else for this PR ?

**Comment by hdep** [comment]
Hi @ghoneycutt  do you still support this module ?

**Comment by ghoneycutt** [comment]
You can help us by reviewing the v3 branch of this repo against any Debian and Ubuntu systems you might have.

**Comment by ghoneycutt** [comment]
We have released a new major version of this module and greatly appreciate your efforts. The new way to support new platforms should be easier as it mostly involves just adding data. Please consider using the new version which supports Puppet 5 and 6 and re-working this PR.

**Pull request closed without merging**